tls/s2n_cipher_preferences.c

```diff
@@ -395,6 +411,7 @@ struct s2n_cipher_suite *cipher_suites_20190214_gcm[] = {
 const struct s2n_cipher_preferences cipher_preferences_20190214_gcm = {
     .count = s2n_array_len(cipher_suites_20190214_gcm),
     .suites = cipher_suites_20190214_gcm,
+    .allow_chacha20_boosting = false
 };
```

Comment:
You need to include a comma on these lines,  or the next person who adds a field will have to modify every line you added